index.html

```diff
@@ -0,0 +1,173 @@
+            <p>1PM-1:45PM</p>
+            <p>
+                I am a Biomimicry Professional and a Visual Communication Designer who is passionate about connecting
+                Nature’s strategies to design. <strong>My research focuses on defining and creating tools for a
+                    life-centered
+                    design thinking methodology that allows the infusion of biomimicry thinking into the traditional
+                    human-centered design process.</strong> I am a professor teaching various multi-disciplinary design
+                studios at
+                The Design School at ASU and also teach the capstone course for the online MS in BMY program. I have
+                presented my work at various conferences including World Usability day, AIGA Y-conference, AIGA PHXDW,
+                SustainbleUX, Audubon society, and Target HQ. My work has been published in the Cooper Hewitt’s Summer
+                2019 Design Journal among other peer reviewed journals.
+            </p>
+        </li>
+        <li>
+            <img src="https://cdn.glitch.com/74171c12-683c-4072-8113-bc7e1b6a35cc%2FSuhailaSikand%20copy.jpg?v=1607029785493"
+                alt="smiling close-up of a dark haired woman with outdoor plants behind her" />
+            <a class="speaker-name" href="http://www.suhaila.me/" target="blank">Suhaila Sikand</a>
+            <p>Climate designer, exploring community, equity, behavior, & design</p>
+            <p>2PM-2:45PM</p>
+            <p>
+                As a social impact and climate designer, I am grounded in community, equity, and sustainability.
+                Unfortunately, human-centered design doesn’t always support these principles with the right intentions.
+                Instead of seeing a problem in a community, let’s get invited to work with the community. Instead of
+                interpreting people’s needs, let’s include them in the process. <strong>Human centered design implies a
+                    surface
+                    level engagement where the power of design remains with the designer, not the community.</strong>
+            </p>
+        </li>
+    </ul>
+    <section id="register">
+        <h2>- register -</h2>
+    </section>
+
+    <form>
+        <ul>
+            <li>
+                <label for="name">name:</label>
+                <input type="text" name="user_name">
+            </li>
+            <li>
+                <label for="mail">email:</label>
+                <input type="email" name="user_email">
+            </li>
+            <li>
+                <label for="phone-number">phone:</label>
+                <input type="phone" name="phone" </li> <li>
+                <label for="msg">question(s):</label>
+                <textarea name="user_message"></textarea>
+            </li>
+            <li class="button">
+                <button class="button-reset" type="submit">send</button>
+            </li>
+        </ul>
```

Comment:
If you find this markup to be excessive, it's not _necessary_ to use `li` around form elements (though I do know it was in the demo MDN code about forms). This would also be valid:

```suggestion
                <label for="name">name:</label>
                <input type="text" name="user_name">
                <label for="mail">email:</label>
                <input type="email" name="user_email">
                <label for="phone-number">phone:</label>
                <input type="phone" name="phone" </li> <li>
                <label for="msg">question(s):</label>
                <textarea name="user_message"></textarea>
                <button class="button-reset" type="submit">send</button>
```

Also, more importantly and _not_ optional, every `input` or `textarea` needs to have an `id` attribute that repeats the value of the `for` attribute of its related label; this is how the browser knows the label and input have a relationship, like so:

```html
                <label for="name">name:</label>
                <input id="name" type="text" name="user_name">

```

In all cases, the `id` must be unique.

Reply:
Thats good to know! I think I'm going to leave the <li> since it helps me read the code better if that's alright